CHANGELOG

```diff
@@ -1,3 +1,12 @@
+Version 1.0.0 (2020-08-27)
+--------------------------
+Reformat code (close #128)
+Update scalafmt to newest (close #127)
+Bump scala-lru-map to 0.5.0 (close #122)
+Bump Circe to 0.13.0 (close #120)
+Replace historic weather requests with current (close #125)
+Update OpenWeatherMap credentials in travis.yml (close #123)
```

Comment:
CHANGELOG should not contain `close`. Given that you're going to update it, please take the opportunity to update the release date.